docs/client.md

```diff
@@ -23,37 +23,39 @@ You can use the full commands (stake) or the short form (s) as shown below.
 
```

Comment:
`create <password>` is a wrong command.
The command is `./razor create` that's all. The password is asked in the next step

Reply:
Also remove the warning after the create command.